test/Parser/classes.js

```diff
@@ -3,10 +3,14 @@ import { vsl, valid, invalid } from '../hooks';
 export default () => describe("Classes", () => {
     valid`class A {}`;
     valid`class A: T {}`;
+    valid`interface A {}`;
+    valid`interface A: T {}`;
```

Comment:
Since this section is titled "Classes" I think it might make more sense to make another test